docs/Style-Guide.md

```diff
@@ -0,0 +1,274 @@
+# Elf Coding Practices Guide 🧝‍♂️
+
+An opintionated styling and best practices guide for React and other typescript based projects. The motivation behind this guide is to provide consenous to writing code that is readable, predictable, and reduces noise so it's easier to understand.
+
+_"Code is read 10x more than it's written."_ - Danny
+
+# Best practices
+
+## Pull Requests
+
+- PRs should be small. The review process grows exponentially the larger the PR. Many smaller PRs > one large PR.
+- Mechanical changes such as large-scale file or variable renaming should not be included in feature PRs.
+- Branch names should include the author name, example `cashd-voting-bug-fix`, to reduce collisions.
+- Since this repo is a monorepo, PR titles should include the app and type of PR. Example `[Gov][Feature] Bulk Voting Vault Delegation`
+
+## Code
+
+- Avoid inline styles, use tailwind or whatever styling framework is chosen for the given project.
+- Avoid abberviations in variable names.
```

Comment:
typo: abbreviations

Might be worth adding the justification for this here too that the reason for this is that people abbreviate inconsistently and it makes it hard to search for things across the codebase.